tests/baselines/reference/pushTypeGetTypeOfAlias.types

```diff
@@ -0,0 +1,29 @@
+=== tests/cases/compiler/foo.js ===
+var bar = require("./bar");
+>bar : { (): void; blah: any; }
+>require("./bar") : { (): void; blah: any; }
+>require : any
+>"./bar" : "./bar"
+
+bar.nonexistentProperty;
+>bar.nonexistentProperty : any
+>bar : { (): void; blah: any; }
+>nonexistentProperty : any
+
+=== tests/cases/compiler/bar.js ===
+module.exports = function () {};
+>module.exports = function () {} : { (): void; blah: any; }
+>module.exports : { (): void; blah: any; }
+>module : { exports: { (): void; blah: any; }; }
+>exports : { (): void; blah: any; }
+>function () {} : () => void
+
+ exports.blah = exports.unknown;
+>exports.blah = exports.unknown : any

```

Comment:
I can't remember how errorType prints in the type baselines. I thought it was `error`. That might be a result of what I noticed in my other comment. (or errorType could actually print as `any`, I don't know.)